explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -27,12 +27,12 @@ import react.semanticui.modules.sidebar.SidebarPusher
 import react.semanticui.modules.sidebar.SidebarWidth
 
 import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportTopLevel
 
 final case class ExploreLayout(
-  c:        RouterCtl[Page],
```

Comment:
Huh? We weren't using this anymore, eh? Good catch.

Reply:
Yeah a bit surprising